README.md

```diff
@@ -9,10 +9,20 @@ This package helps you write better BDD-style tests. It exports three functions
 
 ## Installation
 
+mocha-bdd can be installed via NPM. Since it is a library to help with testing, it should usually be installed as a development dependency, as shown:
+
 ```
 npm install --save-dev --production mocha-bdd
 ```
 
+mocha-bdd is designed to work with [Mocha][mocha]. In particular, it assumes that Mocha’s `describe` and `it` functions are accessible globally (e.g. if your tests are being run using Mocha’s command-line tool). Therefore, Mocha is listed as a peer dependency of mocha-bdd. Mocha can be installed with the following command:
+
+```
+npm install --save-dev --production mocha
+```
+
+The peer dependency listed is Mocha version `^4.0.1`, which is the latest release at the time of writing, and the release which mocha-bdd has been tested against. However, mocha-bdd should work fine with earlier versions of Mocha so long as the `describe` and `it` functions (along with their `.skip` and `.only` methods) are exposed globally.
```

Comment:
Any reason not to use `=` for version here?

Reply:
Yes, it's important this package isn't unnecessarily restrictive because it has to used in combination with other packages and we don't have much control over what other projects are using.

See npm docs: https://docs.npmjs.com/files/package.json#peerdependencies
More explanation: https://nodejs.org/en/blog/npm/peer-dependencies/#using-peer-dependencies